spec.yml

```
@@ -378,10 +378,160 @@ paths:
+  /1.1/get-stats:
+    post:
+      operationId: getStats
+      summary: Returns the stats for an API key and site.
+      description: |
+        Returns the stats for an API key and site associated with that key. The stats consist of total values of spam, ham, missed spam reports, false positive reports, accuracy, a stats breakdown for the requested time period, and the time saved by using Akismet to catch spam.
+
+      parameters:
+        - name: api_key
+          in: query
+          description: The Akismet API key for authorization.
+          required: true
+          schema:
+            type: string
+
+        - name: blog
+          in: query
+          description: The front page or home URL of the instance making the request.
+          required: true
+          schema:
+            type: string
+
+        - name: from
+          in: query
+          description: |
+            The time period for the returned stats. Three values are accepted: 60-days, 6-months, and all. The default value is 6-months.
+
+            A key must have daily stats enabled to receive stats for 60-days. If the key does not have daily stats enabled, the default value of 6-months will be used.
+
+            The response includes a breakdown of stats based on the queried time period (6-months: breakdown by month, 60-days: breakdown by day, all: breakdown by year).
+          required: false
+          schema:
+            type: string
+            enum:
+              - 60-days
+              - 6-months
+              - all
+      responses:
+        '200':
+          description: Get stats response.
+          content:
+            application/json:
+              schema:
+                type: object
+                # OpenAPI Generator doesn't work well with a mixture of properties and additionalProperties,
+                # so have left the response free-form
+                additionalProperties: true
+                example:
+                  'spam': 50
+                  'ham': 137
+                  'missed_spam': 3
+                  'false_positives': 4
+                  'accuracy': 96.25
+                  'breakdown':
+                    '2023-10':
+                      'spam': 30
+                      'ham': 85
+                      'missed_spam': 2
+                      'false_positives': 3
+                      'da': '2023-10-01'
+                    '2023-09':
+                      'spam': 20
+                      'ham': 52
+                      'missed_spam': 1
+                      'false_positives': 1
+                      'da': '2023-09-01'
+                  'time_saved': 3000
+            text/plain:
+              schema:
+                type: string
+              examples:
+                error:
+                  value: "invalid"
+          headers:
+            X-akismet-debug-help:
+              $ref: '#/components/headers/DebugHelpHeader'
+      tags:
+        - key-usage
+  /1.2/get-key-stats:
+    post:
```

```suggestion
    get:
```

I think this is a GET endpoint.